doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc

```diff
@@ -13,14 +13,14 @@ The following procedure demonstrates how to create and use a business rule to ex
 +
 [NOTE]
 ====
-If building the project outside of {CENTRAL}, the following dependencies must be added to the project:
+If you are building the project outside of {CENTRAL}, the following dependencies must be added to the project:
```

Comment:
maybe "add the following dependencies" or "ensure the project includes the following dependencies"?